src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -336,19 +337,44 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
                 }
             }
 
-            foreach (var arg in args)
+            List<ErrorValue> errors = null;
+            for (var i = 0; i < args.Length; i++)
             {
-                if (arg is LambdaFormulaValue lambda)
+                if (args[i] is ErrorValue ev)
+                {
+                    if (errors == null)
+                    {
+                        errors = new List<ErrorValue>();
+                    }
+
+                    errors.Add(ev);
+                }
+                else if (args[i] is BlankValue && _info.ParamTypes[i] is NumberType)
```

Comment:
>BlankValue [](http://example.com/codeflow?start=36&length=10)

There are  lot of cases here to consider (eg, what if I pass Blank to Date, Bool, record ...etc) .

Maybe all those are just failures now, but we should test it. 

In the future,  if we bind to a nullable BooleanValue? , I could imagine that accepts blanks. 

This overlaps the policies that we have in StandardErrorHandling uses:
replaceBlankValues: 
 DoNotReplaceBlank,
 ReplaceBlankWithZero
 ReplaceBlankWithEmptyString